test/jest-e2e.json

```diff
@@ -1,6 +1,7 @@
 {
   "moduleFileExtensions": ["js", "json", "ts"],
   "rootDir": ".",
+  "modulePaths": ["<rootDir>/../"],
   "testEnvironment": "node",
   "testRegex": ".e2e-spec.ts$",
   "transform": {
```

Comment:
Wondering if we can have tests be verbose by default here with `"verbose": true` If we're just testing e2e I imagine having descriptive logging may be helpful (or at least nicer to look at)
<img width="490" alt="image" src="https://github.com/NYCPlanning/ae-zoning-api/assets/7049943/c979f8dd-e3bb-43d6-846f-1cf5fb5a946a">

Reply:
@pratishta could you make a separate ticket for this as well? Something to consider is how updating the default settings will affect test runs when we integrate them into CI workflow. But, I think we should address the question in follow-up work.